xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererGuiTexture.h

```diff
@@ -61,6 +61,9 @@ class CRPRendererGuiTexture : public CRPBaseRenderer
 protected:
   // protected implementation of CRPBaseRenderer
   void RenderInternal(bool clear, uint8_t alpha) override;
+
+private:
+  uint32_t m_vao;
```

Comment:
This should be guarded by a check for `HAS_GL` and `HAS_GLES` otherwise the clang Jenkins job breaks due to the added unused variable warning.